addons/binding/org.openhab.binding.tadoac/src/main/java/org/openhab/binding/tadoac/handler/TadoACHandler.java

```diff
@@ -134,12 +133,11 @@ private synchronized void sendSetting() {
             connector.setSetting(homeid, zoneid, setting);
             updateStatus(ThingStatus.ONLINE);
         } catch (Exception e) {
-            e.printStackTrace();
-            logger.error("could not send state", e);
+            logger.info("could not send state", e);
```

Comment:
Either this is a real error, then this should be logged with `error` severity, or you should assign it the `debug` severity. I would vote for `error` in case the credentials are wrong so the user can do something to fix it. Do you have the option to distinguish `bad credential` from normal connection issues in your library that you use for the connection? Two different `Exception` types would be good.